server/src/main/java/org/opensearch/index/seqno/ReplicationTracker.java

```diff
@@ -1503,6 +1520,11 @@ public synchronized void updateFromClusterManager(
         assert invariant();
     }
 
+    private boolean onRemoteEnabledNode(IndexShardRoutingTable routingTable, String initializingId) {
```

Comment:
We can pass indexSettings as well and then then rename it as `assignedToRemoteStoreNode` .

Reply:
Addressed